internal/http/certificate.go

```diff
@@ -0,0 +1,22 @@
+package http
+
+import (
+	"crypto/x509"
+	"errors"
+	"os"
+)
+
+func LoadRootCAs(path string) (*x509.CertPool, error) {
+	pool, err := x509.SystemCertPool()
+	if err != nil {
+		return nil, err
+	}
+	pemBytes, err := os.ReadFile(path)
+	if err != nil {
+		return nil, err
+	}
+	if ok := pool.AppendCertsFromPEM(pemBytes); !ok {
+		return nil, errors.New("Failed to add certificate authority in file: " + path)
```

Comment:
The error message needs to be revised as this function will not be coupled with root CAs.